src/components/ServiceList.jsx

```diff
@@ -1,22 +1,67 @@
 import { Alert } from "@baltimorecounty/dotgov-components";
 import FilterList from "../components/FilterList";
-import { MostPopularServiceIconStyles } from "../styles";
-import React from "react";
-import ServiceIconLink from "../components/ServiceIconLink";
+import React, { useState } from "react";
+import ServiceIconLink from "./ServiceIconLink";
+import SearchCollapse from "./SearchCollapse";
 import useServices from "../hooks/useServices";
+import {
+  MostPopularServiceIconStyles,
+  LegendText,
+  CollapseContainer
+} from "../styles";
+import Search from "./Search";
 
 const ServiceList = () => {
   const { hasError, serviceItems = [], isLoading } = useServices();
+  const [isChecked, setChecked] = useState(false);
```

Comment:
I think @sgrg1  you made this change before, but saying this isMostPopular checked or somethign like that makes it easier to know what is checked. This is important if we add more filters.

Reply:
must have gotten over written from the previous branch? Not clue but its been updated again.